ModuleSystem/module_scripts_common_warp.py

```diff
@@ -0,0 +1,764 @@
+from header_common import *
+from header_operations import *
+from module_constants import *
+from header_parties import *
+from header_skills import *
+from header_mission_templates import *
+from header_items import *
+from header_triggers import *
+from header_terrain_types import *
+from header_music import *
+from ID_animations import *
+from ID_troops import *
+from ID_factions import *
+#from module_troops import *
+
+from module_info import wb_compile_switch as is_a_wb_script
+
+# Credits to Autolykos, thread https://forums.taleworlds.com/index.php?topic=335511.0
```

Comment:
@ehsn That's correct, and every slot allocation stays in the savegame forever no matter how you try to blank it out. Good thinking on your part reusing as much space as possible.

Rediscovering the callback subsystem and later accommodating the code for the mod needs is impressive to say the least. I also like the fancy garbage collector and savegame versioning you introduced.